pkg/controllers/syncer/translator/namespaced_translator.go

```diff
@@ -160,3 +166,12 @@ func (n *namespacedTranslator) TranslateMetadata(vObj client.Object) client.Obje
 func (n *namespacedTranslator) TranslateMetadataUpdate(vObj client.Object, pObj client.Object) (bool, map[string]string, map[string]string) {
 	return translate.Default.ApplyMetadataUpdate(vObj, pObj, n.syncedLabels, n.excludedAnnotations...)
 }
+
+func ConvertLabelKey(key string) string {
+	return ConvertLabelKeyWithPrefix(LabelPrefix, key)
+}
+
+func ConvertLabelKeyWithPrefix(prefix, key string) string {
```

Comment:
Please get rid of this as this couldn't be used correctly with multi-namespace syncer as we don't want to replace the labels there